git_machete/handmade_short_docs.py

```diff
@@ -0,0 +1,35 @@
+from typing import Dict
+
+# ---------------------------------------------------------------------------------------------------------
+# Warning: This file is supposed to be modified manually
```

Comment:
Not a warning ;) just
```suggestion
# This file is supposed to be modified manually
```
Warnings make sense where the file is NOT supposed to be edited manually